src/index.html

```diff
@@ -4,9 +4,441 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__moyo-logo"
+          src="images\logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__link" href="#laptops_computers">laptops & computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            src="images\imac.jpeg"
+            alt="imac image"
+            class="card__image"
+          >
+
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <h3 class="card__product-code">
+            Product code: 195434
+          </h3>
+
+          <div class="stars stars--4">
+            <ul class="stars__rate-stars">
+              <li class="stars__rate-stars-star"></li>
+              <li class="stars__rate-stars-star"></li>
+              <li class="stars__rate-stars-star"></li>
+              <li class="stars__rate-stars-star"></li>
+              <li class="stars__rate-stars-star"></li>
+            </ul>
+
+            <p class="stars__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            href="#buy"
+            class="card__buy-button"
+            data-qa="card-hover"
+          >
+          buy
+          </a>
```

Comment:
Check correct indents for the content
```suggestion
          >
            buy
          </a>
```